### PR TITLE
Gh367

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/changes.txt [hunk 1]
```diff
@@ -6,6 +7,12 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2022-01-25  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltplain.dtx (section{Plain \TeX}):
+	Alter \obeyspaces and \obeylines to use a redirection for special special
```
Comment (muzimuzhi) [inline base/changes.txt:13]: word "special" is duplicated

File: base/doc/ltnews35.tex [hunk 2]
```diff
@@ -323,6 +323,38 @@ \subsection{???}
...
+hardwired solution did before.
+
+\begin{quote}
+  \renewcommand\obeyedspace{\ \textbullet\ }
+  \footnotesize\obeyspaces%
+But this means that it is now possible to %
+achieve special effects in a safe way. %
+This paragraph, for example, was produced by %
+making \cs{obeyedspace} generate %
+\texttt{\cs{hspace}\{\cs{\verbvisiblespace}\cs{textbullet}\cs{\verbvisiblespace}\}} and %
```
Comment (PhelypeOleinik) [inline base/doc/ltnews35.tex:349]: No `\cs{hspace}`: `\texttt{\cs{\verbvisiblespace}\cs{textbullet}\cs{\verbvisiblespace}}`

File: base/ltplain.dtx [hunk 3]
```diff
@@ -1186,18 +1186,143 @@
...
+%<latexrelease>
+%<latexrelease>\let\obeyedline\@undefined
+%<latexrelease>\let\obeyedspace\@undefined
+%<latexrelease>\EndIncludeInRelease
+%<*2ekernel>
+%    \end{macrocode}
+%
+%    An active space generates \cs{space} by default (for example in a \cs{write}):  
+%    \begin{macrocode}
+%    \end{macrocode}
```
Comment (PhelypeOleinik) [inline base/ltplain.dtx:1325]: Stray lines?

**Review by PhelypeOleinik — APPROVED** [review]

File: base/changes.txt (hunk 1 above)
Comment (FrankMittelbach) [reply base/changes.txt:13]: thanks

File: base/doc/ltnews35.tex (hunk 2 above)
Comment (FrankMittelbach) [reply base/doc/ltnews35.tex:349]: yeah a leftover from my earlier example

File: base/ltplain.dtx (hunk 3 above)
Comment (FrankMittelbach) [reply base/ltplain.dtx:1325]: well, text partly but not fully reordered

**Review by davidcarlisle — APPROVED** [review]

**Pull request merged**